tests/scr-readingsystem-support_iframe_svg/OPS/iframe_content.svg

```diff
@@ -0,0 +1,80 @@
+  <script>//<![CDATA[
+      /**
+      * A simple test that proves that scripting is enabled.
+      */ 
+      function test_support() {
+        const report = document.getElementById('middle_line');
+        report.replaceChildren(document.createTextNode('supports scripting in SVG.'))
+
+        const passes = document.getElementById('passes');
+        passes.replaceChildren(document.createTextNode(''))
+
+        try {
+          const ers = navigator.epubReadingSystem;
+          // For code testing
+          // const ers = {
+          //  name : "Test ERS setup",
+          //  version: "0.0.0"
+          // };
+          if (ers !== undefined) {
+            const ers_passes = document.getElementById('ers_passes');
+            const ers_fail = document.getElementById('ers_fails');
+            const ers_name = document.getElementById('ers_name');
+            const ers_version = document.getElementById('ers_version');
+
+            ers_name.replaceChildren(document.createTextNode(ers.name));
+            ers_version.replaceChildren(document.createTextNode(ers.version));
+
+            ers_fail.style.display = 'none';
+            ers_passes.style.display = 'block';
+
+          }
+        } catch (e) {
+          // console.log(e);
+          // Let the original, negative display stay as is. 
+        }
+      }
+      window.addEventListener('load', test_support);
+      //]]>
+  </script>
+  <style>
+    .ers { font: bold 5px monospace; }
+    text {
+      font-size: 5px;
+      text-anchor: middle;
+      fill: white;
+    }
+  </style>
+  <rect width="100%" height="100%" fill="red" />
+
+  <g transform="translate(0,-50)">
+    <ellipse cx="150" cy="100" rx="60" ry="30" fill="green" />
+    <g transform="translate(0,-30)">
+      <text x="150" y="125">The reading system</text>
+      <text x="150" y="132" id="middle_line">does not support scripting in SVG.</text>
+      <text x="150" y="139" id="passes">Test fails.</text>
+    </g>
+  </g>
+
+  <g transform="translate(0, 50)" style="display:block" id="ers_fails">
+    <ellipse cx="150" cy="100" rx="60" ry="30" fill="green" />
+    <g transform="translate(0,-30)">
+      <text x="150" y="125">The reading system</text>
+      <text x="150" y="132">does not implement the <tspan class="ers">epubReadingSystem</tspan> object.</text>
+      <text x="150" y="139">Test fails.</text>
+    </g>
+  </g>
+
+  <g transform="translate(0, 50)" style="display:none" id="ers_passes">
+    <ellipse cx="150" cy="100" rx="60" ry="30" fill="green" />
+    <g transform="translate(0,-38)">
+      <text x="150" y="125">The reading system</text>
+      <text x="150" y="132">implements the <tspan class="ers">epubReadingSystem</tspan> object.</text>
+      <text x="150" y="139">Reading System Name: <tspan id="ers_name">sg</tspan></text>
+      <text x="150" y="146">Reading System Version: <tspan id="ers_version">1.0.0</tspan></text>
```

Comment:
Why do these two lines have hardcoded values?

Reply:
These were just phony values that were used _testing_ the SVG. The enclosed `<g>` is, by default, not displayed; it is only displayed if scripting is enabled in the first place and the values are changed by the real ones (see lines 26 and 27 in the script). But, just to avoid any problems, I have removed these.